inst/NEWS

```diff
@@ -4,4 +4,4 @@ Changes for v0.99
 - compliance for bioconductor
 - qza_to_phyloseq is less strict about objects included in phyloseq object
 - added vignette and help
-- the function qza_to_tse has been added, facilitating operability between qiime2R and microbiome packages
+- the function qza_to_tse has been added, facilitating operability between qiime2R and R/Bioc packages suporting TreeSummarizedExperiment
```

Comment:
suporting -> supporting

operability -> interoperability

remove "has been" to keep it short

split to two lines

Reply:
should also be fixed now